### ISSUE TITLE
Questions about training detectors

### ISSUE THREAD

**Issue by linzhi123** [sergiomsilva/alpr-unconstrained#26]
Excuse me, when training detectors, can only one license plate be used in a picture?

**Comment by danishansari** [comment]
I think all should be labeled correctly, and it should work fine.

**Comment by wmn931201** [comment]
Do you have the train images which include AOLP,SSIG and Cars? Thank you very much if you can provide me ! @linzhi123

**Comment by sergiomsilva** [comment]
@linzhi123 yes, in fact we are using only one per image to train the detector.

**Comment by linzhi123** [comment]
I don't have these images @wmn931201

**Comment by E-Burger** [comment]
@linzhi123 you can download images at https://ai.stanford.edu/~jkrause/cars/car_dataset.html for the cars dataset. The annotations is available at http://www.inf.ufrgs.br/~crjung/alpr-datasets/. The format of the data is: Number of corners, top_left_x, top_right_x, bottom_right_x, bottom_left_x, top_left_y, top_right_y, bottom_right_y, bottom_left_y, label. The label can be LP for license plate or the specific license plate characters. You will notice the coordinates is between 0 and 1, you have to multiply it by the original image width for x values and height for y values to get the correct coordinates. Try and load a image and plot the boundary box

**Comment by zongoalbert** [comment]
Hello ,
Could you please tell me how can be detect the arabic lettres , im working at the detection for Moroccan plate ,
when i run your code the detection of plates is well but can't read arabic letter in the middle.
exemple :
image

Edit: This is Example of Arabic letter in the middle of plate  : أ ب ت ج ح د هـ The expected result is: د
image
The vertical lines are irrelevant, I have to use them to separate the image and read data separately.

**Comment by zongoalbert** [comment]
please, need your help
